src/Tribe/Views/V2/Customizer.php

```diff
@@ -32,6 +32,55 @@ public function filter_sections( array $sections, $customizer ) {
 		return $sections;
 	}
 
+	public function include_global_elements_settings( $section, $manager, $customizer ) {
+		// Event Title.
+		$manager->add_setting(
+			$customizer->get_setting_name( 'event_title_color', $section ),
+			[
+				'default'              => '#141827',
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new \WP_Customize_Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_title_color', $section ),
+				[
+					'label'    => esc_html__( 'Event Title', 'the-events-calendar' ),
+					'section'  => $section->id,
+					'priority' => 8,
+				]
+			)
+		);
+
+		// Event Date & Time.
+		$manager->add_setting(
+			$customizer->get_setting_name( 'event_date_time_color', $section ),
+			[
+				'default'              => '#141827',
+				'type'                 => 'option',
+				'sanitize_callback'    => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			]
+		);
+
+		$manager->add_control(
+			new \WP_Customize_Color_Control(
+				$manager,
+				$customizer->get_setting_name( 'event_date_time_color', $section ),
+				[
+					'label'       => esc_html__( 'Event Date and Time', 'the-events-calendar' ),
+					'description' => esc_html__( 'Main date and time display on views and single event pages.', 'the-events-calendar' ),
+					'section'     => $section->id,
+					'priority' => 8,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 4 space(s) between "'priority'" and double arrow, but found 1.

<sub>hash: 64bcc91a53b41c9c135527e88034c6e4</sub>